float32 x_cap
float32 y_cap
float32 z_cap
float32 x
float32 y
float32 z
int8 h_cap
uint8 s_cap
uint8 i_cap
int8 h
uint8 s
uint8 i
bool precise
---
int8 status
bool prior_setting
